airgun/views/virtwho_configure.py

```diff
@@ -124,7 +124,7 @@ class VirtwhoConfigureCreateView(BaseLoggedInView):
     satellite_url = TextInput(id='foreman_virt_who_configure_config_satellite_url')
     hypervisor_id = FilteredDropdown(id='foreman_virt_who_configure_config_hypervisor_id')
     debug = Checkbox(id='foreman_virt_who_configure_config_debug')
-    proxy = TextInput(id='foreman_virt_who_configure_config_proxy')
+    proxy = FilteredDropdown(id='foreman_virt_who_configure_config_http_proxy_id')
```

Comment:
@hkx303 should we change it's name to http_proxy to be more precise?

Reply:
Hey Jameer, thanks for your advice. Actually, we all use `proxy` as the name of this option, such as API:
`foreman_virt_who_configure_config[proxy]: optional , nil allowed` :)